aspnetcore/blazor/components.md

```diff
@@ -575,13 +575,16 @@ Razor directives are shown in the following table.
 
 | Directive | Description |
 | --------- | ----------- |
+| [\@code](xref:mvc/views/razor#section-5) | Adds a C# code block to a component. |
```

Comment:
I might mention here that this is an alias for `@functions`. We don't currently have any plans to differentiate their feature set.